lib/rihanna.ex

```diff
@@ -85,6 +91,87 @@ defmodule Rihanna do
     raise ArgumentError, @enqueue_help_message
   end
 
+  @schedule_help_message """
+  Rihanna.schedule requires either two arguments in the form {mod, fun, args}
+  and schedule options or three arguments including a module implementing
+  Rihanna.Job, its args, and schedule options.
+
+  Schedule options require either an `at:` DateTime struct or `in:` duration
+  in milliseconds.
+
+  For example, to run IO.puts("Hello") after midday on 1st July, 2018 (UTC):
+
+  > due_at = DateTime.from_naive!(~N[2018-07-01 12:00:00], "Etc/UTC")
+  > Rihanna.schedule({IO, :puts, ["Hello"]}, at: due_at)
+
+  Or, if you have a job called MyJob that implements the Rihanna.Job behaviour:
+
+  > Rihanna.schedule(MyJob, arg, at: due_at)
+
+  To run IO.puts("Hello") in one hour:
+
+  Rihanna.schedule({IO, :puts, ["Hello"]}, in: :timer.hours(1))
```

Comment:
Be sure to indent these examples by four spaces to give proper formatting in exdocs.

Reply:
As above, this text is part of the `ArgumentError` help message.